.changeset/cuddly-insects-pump.md

```diff
@@ -0,0 +1,8 @@
+---
+"@aws-amplify/ui-react": major
+"@aws-amplify/ui": major
+"@aws-amplify/ui-angular": major
+"@aws-amplify/ui-vue": major
+---
+
+**BREAKING**: Bump minimum `aws-amplify` peer dependency version to `4.3.29`. Please see the official instruction to upgrade Amplify dependencies: [docs](https://docs.amplify.aws/lib/troubleshooting/upgrading/q/platform/js/#check-for-duplicate-versions)
```

Comment:
This changeset isn't necessary anymore since we will be requiring `aws-amplify` v5 anyway.

Reply:
Oh yeah, good catch, and we should add a changeset for `handleSignUp` changes too.